### ISSUE TITLE
"make test" after installation passes every test but "test_sqrt.py:146"

### ISSUE THREAD

**Issue by realisation** [ethereum/vyper#1506]
```
================================================================== FAILURES ===================================================================
____________________________________________________________ test_sqrt_valid_range ____________________________________________________________
[gw0] darwin -- Python 3.7.3 /Users/realisation/vyper-venv/bin/python

sqrt_contract = <tests.base_conftest.VyperContract object at 0x1127d2400>

    @hypothesis.given(
>       value=hypothesis.strategies.decimals(
            min_value=Decimal(0),
            max_value=Decimal(SizeLimits.MAXNUM),
            places=DECIMAL_PLACES
        )
    )
    @hypothesis.example(Decimal(SizeLimits.MAXNUM))
    @hypothesis.example(Decimal(0))
    @hypothesis.settings(
        deadline=400,
    )
    def test_sqrt_valid_range(sqrt_contract, value):

f          = <function given.<locals>.run_test_with_generator.<locals>.wrapped_test at 0x1124a7ae8>
sqrt_contract = <tests.base_conftest.VyperContract object at 0x1127d2400>

tests/parser/types/numbers/test_sqrt.py:146: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
.eggs/hypothesis-4.11.7-py3.7.egg/hypothesis/core.py:593: in execute
    % (test.__name__, text_repr[0])
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <hypothesis.core.StateForActualGivenExecution object at 0x1127d2eb8>
message = "Hypothesis test_sqrt_valid_range(sqrt_contract=<tests.base_conftest.VyperContract at 0x1127d2400>, value=Decimal('669...460715481794508.4517941248')) produces unreliable results: Falsified on the first call but did not on a subsequent one"

    def __flaky(self, message):
        if len(self.falsifying_examples) <= 1:
>           raise Flaky(message)
E           hypothesis.errors.Flaky: Hypothesis test_sqrt_valid_range(sqrt_contract=<tests.base_conftest.VyperContract at 0x1127d2400>, value=Decimal('6690223559559186942460715481794508.4517941248')) produces unreliable results: Falsified on the first call but did not on a subsequent one

message    = "Hypothesis test_sqrt_valid_range(sqrt_contract=<tests.base_conftest.VyperContract at 0x1127d2400>, value=Decimal('669...460715481794508.4517941248')) produces unreliable results: Falsified on the first call but did not on a subsequent one"
self       = <hypothesis.core.StateForActualGivenExecution object at 0x1127d2eb8>

.eggs/hypothesis-4.11.7-py3.7.egg/hypothesis/core.py:761: Flaky
----------------------------------------------------------------- Hypothesis ------------------------------------------------------------------
Falsifying example: test_sqrt_valid_range(sqrt_contract=<tests.base_conftest.VyperContract at 0x1127d2400>, value=Decimal('6690223559559186942460715481794508.4517941248'))
Unreliable test timings! On an initial run, this test took 523.00ms, which exceeded the deadline of 400.00ms, but on a subsequent run it took 280.70 ms, which did not. If you expect this sort of variability in your test timings, consider turning deadlines off for this test by setting deadline=None.

You can reproduce this example by temporarily adding @reproduce_failure('4.11.7', b'AXicY2BgYGbAAAAATwAE') as a decorator on your test case
```

**Comment by realisation** [comment]
This was after cloning the repository and running `make`.

**Comment by siraben** [comment]
I can reproduce this bug sometimes when I run the test suite in a Docker instance. @jacqueswww perhaps increase the deadline in https://github.com/ethereum/vyper/blob/929a33b3ca5ba8505c51fe96c8f7378977c3c5a3/tests/parser/types/numbers/test_sqrt.py#L155 ?

**Comment by iamdefinitelyahuman** [comment]
I think this was fixed in #1571 and so this issue can be closed.
